Content/Application/ContentDataMapper/DataMapper/RoutableDataMapper.php

```
@@ -147,44 +135,43 @@ public function map(
         }
 
         if (null === $entityClass || null === $routeSchema) {
-            // TODO FIXME add test case for this
-            return; // @codeCoverageIgnore
+            return;
```

i think this is most likely an error (eg forget to add configuration). i think i would throw an exception here too?